### PR TITLE
Deposit fee feature (closes #886)

### LINKED ISSUE #886: Deposit fee feature

### PR DESCRIPTION
**Pull request by ysv**

### CODE REVIEW

File: app/models/concerns/fee_chargeable.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module FeeChargeable
+  extend ActiveSupport::Concern
+
+  included do
+    before_validation(on: :create) { calc_fee }
+
+    validates :fee, numericality: { greater_than_or_equal_to: 0 }
```
Comment (yivo) [inline app/models/concerns/fee_chargeable.rb]: @yssavchuk You can combine these two in single statement: `presence: true, numericality: { greater_than_or_equal_to: 0 }`

File: spec/models/withdraw_spec.rb [hunk 2]
```diff
@@ -230,6 +230,19 @@
...
   end
 
+  context 'when withdraw fee is fixed with value 10' do
+    let(:withdraw) { create(:usd_withdraw, sum: 200.to_d) }
+
+    before do
+      Currency.any_instance.expects(:withdraw_fee).once.returns(10)
+    end
+
+    it 'should compute fee' do
```
Comment (yivo) [inline spec/models/withdraw_spec.rb]: @yssavchuk Can you exclude `should` from msg? `it 'computes fee'`.

**Review by yivo — CHANGES_REQUESTED** [review]

**Comment by yivo** [comment]
@yssavchuk Can you also check if you have updated admin panel with this fee field?

**Comment by ysv** [comment]
@yivo Rebased
Also I've deleted active yaml base in this pr because I've forgot about it

**Review by yivo — APPROVED** [review]

**Pull request merged**